test/acceptance/tests/basic_test.go

```diff
@@ -55,7 +60,9 @@ func TestBasic(t *testing.T) {
 			partition := ""
 			queryString := ""
 			tfVars["consul_image"] = "public.ecr.aws/hashicorp/consul:1.15.1"
```

Comment:
We should update this ref and the `consul-enterprise` ref below to the latest Consul 1.16.1 release.

Reply:
done

Reply:
We also need to update the version for the consul-enterprise image: https://github.com/hashicorp/terraform-aws-consul-lambda/blob/ee0eaeb921c99b12b41e5b7fe8bc4120cafb9267/test/acceptance/tests/basic_test.go#L79